<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{background-color: rgba(150, 170, 255, 1);border-radius: 2px;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MenuBtn" name="menubtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,128,206, 0);border-width:2px; border-style:solid; border-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Menu" name="menu" native="true">
    <property name="geometry">
     <rect>
      <x>-200</x>
      <y>60</y>
      <width>200</width>
      <height>561</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(30,84,156, 1);</string>
    </property>
   </widget>
   <widget class="WindowControl" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../project_mojidict_msvc/logo.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWebEngineView" name="webEngineView" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>60</y>
      <width>1364</width>
      <height>707</height>
     </rect>
    </property>
    <property name="url" stdset="0">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="titlebar" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(68,114,196,1); border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="SearchBox" name="searchBox" native="true">
    <property name="geometry">
     <rect>
      <x>483</x>
      <y>10</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>menubtn</zorder>
   <zorder>closeBtn</zorder>
   <zorder>icon</zorder>
   <zorder>webEngineView</zorder>
   <zorder>menu</zorder>
   <zorder>searchBox</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MenuBtn</class>
   <extends>QLabel</extends>
   <header location="global">menubtn.h</header>
  </customwidget>
  <customwidget>
   <class>Menu</class>
   <extends>QWidget</extends>
   <header location="global">menu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WindowControl</class>
   <extends>QLabel</extends>
   <header location="global">windowcontrol.h</header>
  </customwidget>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
  <customwidget>
   <class>titlebar</class>
   <extends>QLabel</extends>
   <header location="global">titlebar.h</header>
  </customwidget>
  <customwidget>
   <class>SearchBox</class>
   <extends>QWidget</extends>
   <header location="global">searchbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
